04-materials/02-anatomy-of-extensions.md

```diff
@@ -52,6 +52,9 @@ This tutorial is inspired by many prior works.
       photo API is not reliable during the current government shutdown 😭)
     * Remove unnecessary use of `conda` in favor of more standard tooling
     * Structure the activities around in person teaching and exercises
+  * [Agentic AI Programming for Python Course](https://training.talkpython.fm/courses/agentic-ai-programming-for-python)
+    by TalkPython['Training']
+    * Ideas for teaching AI-assisted coding in 2025
```

Comment:
Let's move this into a similar "Inspired by..." block in module 6 instead of module 2? Or should we have one dedicated page for our inspirations?